corehq/apps/translations/app_translations.py

```diff
@@ -650,6 +646,7 @@ def update_form_translations(sheet, rows, app):
     module_index = int(mod_text.replace("module", "")) - 1
     form_index = int(form_text.replace("form", "")) - 1
     form = app.get_module(module_index).get_form(form_index)
+    rows = get_unicode_dicts(sheet)  # fetch once, because sheet.__iter__ can only be called once
```

Comment:
curious how this still works since it seems like `get_unicode_dicts(sheet)` now gets called multiple times?

Reply:
Not following this question - `get_unicode_dicts` is only called once on each sheet.

Reply:
> `get_unicode_dicts` is only called once on each sheet

This answers the question - didn't realize that.